### PR TITLE
Subscriptions Resource Missing Calls

### PR DESCRIPTION
**Pull request by wwjfrodriguez**
Ref  [killbill-client-ruby Issue #36](https://github.com/killbill/killbill-client-ruby/issues/36)

### CODE REVIEW

File: lib/killbill_client/models/subscription.rb [hunk 1]
```diff
@@ -132,6 +132,55 @@ def update_bcd(user = nil, reason = nil, comment = nil, effective_from_date = ni
       end
 
 
+      #
+      # Block a Subscription
+      #
+      def block_subscription(subscription_id, state_name, service, block_change, block_entitlement, block_billing, requested_date = nil, user = nil, reason = nil, comment = nil, options = {})
```
Comment (pierre) [inline lib/killbill_client/models/subscription.rb]: 1. No need to pass a `subscription_id` (it's an instance method)
2. Let's rename it to `set_blocking_state` (symmetry with `account.rb` and `bundle.rb`)

File: lib/killbill_client/models/subscription.rb [hunk 2]
```diff
@@ -132,6 +132,55 @@ def update_bcd(user = nil, reason = nil, comment = nil, effective_from_date = ni
...
+                           :user    => user,
+                           :reason  => reason,
+                           :comment => comment,
+                       }.merge(options)
+      end
+
+      #
+      # Create an entitlement with addOn products
+      #
+      def create_entitlement_with_addOn(entitlements,requested_date, entitlement_date, billing_date, migrated = false, call_completion = false, callTimeoutSec = 3, user = nil, reason = nil, comment = nil, options = {})
```
Comment (pierre) [inline lib/killbill_client/models/subscription.rb]: This is already implemented in `lib//killbill_client/models/bulk_subscription.rb`. Easiest to check for these is to `grep` the path suffix (e.g. `createEntitlementWithAddOns` in this case).

Comment (wwjfrodriguez) [reply lib/killbill_client/models/subscription.rb]: There two different API calls named very similar for this:  createEntitlementsWithAddOns (Entitlements with 's') and  createEntitlementWithAddOns (Entitlement without 's'). The second one was missed.

Comment (pierre) [reply lib/killbill_client/models/subscription.rb]: Oops, indeed! 👍

**Pull request merged**